dodiscover/replearning/gin.py

```diff
@@ -51,8 +87,21 @@ def _extract_edgelists(adj_mat, names):
         return pdag
 
 
-    def fit(self, data, context):
+    def fit(self, data: 'DataFrame', context: 'DataFrame'):
```

Comment:
a context is an instance of the [Context class.](https://github.com/py-why/dodiscover/blob/main/dodiscover/context.py#L9)